src/index.ts

```diff
@@ -54,6 +56,34 @@ function getRecordExtractorSource(): string {
 }`;
 }
 
+function addComment(crawlerId: string): void {
+  try {
+    const pathArray = CRAWLER_API_BASE_URL.split('/');
+    const protocol = pathArray[0];
+    const host = pathArray[2];
+    const baseUrl = `${protocol}//${host}`;
+
+    const message = `<p>Check your created <a href="${baseUrl}/admin/crawlers/${crawlerId}/overview" target="_blank">Crawler</a></p>
+    <p>Check your created index on your <a href="https://www.algolia.com/apps/${ALGOLIA_APP_ID}/explorer/browse/${CRAWLER_NAME}" target="_blank">Algolia Application</a></p>`;
+
+    const context = github.context;
+    if (context.payload.pull_request === undefined) {
+      core.info('No pull request found.');
+      return;
+    }
+    const prNumber = context.payload.pull_request.number;
+
+    const octokit = github.getOctokit(GITHUB_TOKEN);
+    octokit.rest.issues.createComment({
```

Comment:
Now that I think about it, would be create to not recreate a comment but find the previous one
can be done in an other PR